lib/virt_autotest/common.pm

```diff
@@ -279,6 +279,10 @@ if (get_var("REGRESSION", '') =~ /xen/) {
             name => 'sles15sp3',
             ip => 'd215.qam.suse.de',
         },
+        sles15sp4 => {
+            name => 'sles15sp4',
+            ip => 'dhcp63.qa.suse.cz',
```

Comment:
@tbaev  Hello Teodor, a quick question. Is this ip/domain_name mapped in DHCP server for the guests' mac address, so that after each reboot, the guest can always get the same ip?

Reply:
It is not, I will do that.

Reply:
Look good to me!   
One thing should be noted that static ip is not needed here as the vmware and hyperv test can get guest's ip using a command from hypervisor

Reply:
Okay, if that is the case I will remove all the IP settings for our guests, run a test to verify and edit my PR.

Reply:
@tbaev  Is this PR done for changes? When the request for static ip is merged, please ping me to merge it if it is ready. 
Btw, would you please double confirm if HYPERV guest behavior(static ip needed) is correct? If no, would you please open a ticket in our backlog? We can arrange fix later.

Reply:
We are still waiting for the PR https://gitlab.suse.de/OPS-Service/salt/-/merge_requests/2324 

I have double checked and yes, hyperV is not working without IP address.

I have made the final change to this PR with the static IP settings, so technically when the github and gitlab pull request are merged everything should work.

Reply:
@alice-suse 
The request for static ip is merged and working for the guest, we can finish with merging this PR too. :)

Reply:
Cool~, merge it now.